app/src/main/java/com/example/workoutapp/RecordActivity.kt

```diff
@@ -30,15 +30,15 @@ class RecordActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    RecordWO()
```

Comment:
This was more unclear to me, WO could mean different things like Work Order, so changed it to Workout

Reply:
Agreed.

Reply:
Avoid using abbreviations.  You'll waste more time trying to remember what abbreviation you used, than you'll save in typing fewer letters.